src/shared_utilities/drivers.jl

```diff
@@ -304,6 +304,11 @@ Computes turbulent surface fluxes at a point on a surface given
 
 This returns an energy flux and a liquid water volume flux, stored in
 a tuple with self explanatory keys.
+
+Please note that this function, if r_sfc is set to zero, makes no alteration
```

Comment:
Maybe we could explain what it does if `r_sfc !=0` too

Reply:
I am trying to work too quickly today and being a little sloppy - merged before I saw this. I'll fix this in my next PR.